service/src/main/java/uk/nhs/adaptors/gp2gp/mhs/model/Identifier.java

```diff
@@ -1,10 +1,14 @@
 package uk.nhs.adaptors.gp2gp.mhs.model;
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Builder
 @Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
What are all these about? You make no mention of why this is important in the PR. Without explaining why you've made a change it makes it very hard for other people on the team to understand what you're thinking.

Reply:
When I defined objectMapper as bean for the whole application it asked for default constructor in Identifier.java as without it it throws `com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `uk.nhs.adaptors.gp2gp.mhs.model.Identifier`` exception hence it added this default constructor.